.github/workflows/ci.yml

```diff
@@ -15,6 +15,45 @@ concurrency:
   cancel-in-progress: ${{ github.ref != format('refs/heads/{0}', 'main') }}
 
 jobs:
+  bump:
+    runs-on: ubuntu-latest
+    env:
+      GITHUB_TOKEN: ${{ secrets.CONDUKTORBOT_GHCR_RW }}
```

Comment:
@kimpiro I guess I need your help here to make sure the `CONDUKTORBOT_GHCR_RW` is available.